opal/scaffolding/plugin_scaffold/runtests.py.jinja2

```diff
@@ -34,6 +34,7 @@ settings.configure(DEBUG=True,
                                    'django.contrib.contenttypes',
                                    'django.contrib.sessions',
                                    'django.contrib.staticfiles',
+                                   'django.contrib.messages',
```

Comment:
Don't we have to do this step in all applications & thus should be in the upgrading guide?

Reply:
so the change from django is that admin now requires messages, so run tests required messages.

The application already scaffolds a settings.py that includes messages so you should not need to add it to the upgrading guide because its already there.

We can do if you think that some people may have removed it manually and now need to add it again?